extensions/sql-database-projects/package.json

```diff
@@ -43,7 +43,7 @@
       {
         "title": "%sqlDatabaseProjects.Settings%",
         "properties": {
-          "sqlDatabaseProjects.netCoreSDKLocation": {
+          "sqlDatabaseProjects.netCoreSDK Location": {
```

Comment:
Did you verify this still works? I'm a little surprised that spaces in there are allowed.

At the very least don't you need to update the string in netcoreTools.ts?

Reply:
oops forgot to run the tests before pushing. Fixed now and also manually verified it works. Yeah I was also surprised that this worked since it looks like it uses camel case to figure out where the spaces go.